### DISCUSSION TITLE
MSTAR SSC328Q

### DISCUSSION THREAD

**Discussion by StealUrKill** [linux-chenxing/linux-chenxing.org#7] [General]
Have you seen any more info on this particular SOC? I just received a 1k units and the site you listed says discontinued. 

![IMG_20210107_175912](https://user-images.githubusercontent.com/35749471/103958280-3a0b1c80-5112-11eb-96a0-a098d9eea376.jpg)

**Comment by fifteenhex** [comment]
Hi,

I don't have anything specific about the SSC328Q except that it seems to be in the "infinity5" family.
The MC800L module seems to be discontinued but I'm not sure if the chip itself is.

The date code on the chip you have looks like 20<something><something>? If so that should mean it was made in 2020 so I would be surprised if it was discontinued.

Anyhow.. I think that chip is probably supported in one of the vendor kernels we have. So if you want to customise the kernel you should be able to do that. I think I've booted my kernel on that chip once but it wouldn't be very useful to you as the camera and video encoder aren't supported at all.

Would you mind adding an item for your device in the "known devices" section here: https://linux-chenxing.org/infinity5/

Thanks,

Daniel
